### PR TITLE
[GraphQL] Security

### PR DESCRIPTION
**Pull request by alanpoulain**

### CODE REVIEW

File: core/graphql.md [hunk 1]
```diff
@@ -68,3 +72,42 @@ class Offer
...
 ```
+
+## Security (access_control)
+
+To add a security layer to your queries and mutations, follow the [security](security.md) documentation.
+
+If your security needs differ between REST and GraphQL, add the particular parts in the `graphql` key.
+
+In the example below, we want the same security rules as in REST, but we also want to allow an admin to delete a book in GraphQL only.
+Please note it's not possible to update a book in GraphQL because the operation is not defined.
```
Comment (soyuka) [inline core/graphql.md]: Please note that

File: core/graphql.md [hunk 2]
```diff
@@ -59,7 +60,10 @@ use ApiPlatform\Core\Annotation\ApiResource;
  *     graphql={
  *         "query"={
  *              "filters"={"offer.date_filter"}
- *          }
+ *          },
+ *          "delete"={},
```
Comment (dunglas) [inline core/graphql.md]: The syntax `"delete", "update", "create"` should work IIRC.

File: core/graphql.md [hunk 3]
```diff
@@ -68,3 +72,42 @@ class Offer
     // ...
 }
 ```
+
+## Security (access_control)
```
Comment (dunglas) [inline core/graphql.md]: \`access_control\`

**Comment by alanpoulain** [comment]
Fixed.

**Comment by soyuka** [comment]
thanks @alanpoulain

**Pull request merged**